src/validation/validation.ts

```diff
@@ -1,8 +1,5 @@
 import { record, z } from 'zod';
-import type {
-  SerializedNestedType,
-  SerializedType,
-} from './serialize-provable.ts';
```

Comment:
now we have these types twice, can you create a third file `serialize-provable-types.ts` and import them from there in both of the other files?

Reply:
(assuming that importing them from the original file causes an unwanted o1js dependency)